board/app/models/thread.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+class Thread extends AppModel{
+
+//Minimum Length
+const MIN_TITLE_LENGTH = 1;
+
+//Maximum Length
+const MAX_TITLE_LENGTH = 30;
+
+
+	public $validation = array(
+		'title' => array(
+			'length' => array(
+				'validate_between', self::MIN_TITLE_LENGTH, self::MAX_TITLE_LENGTH),
+			),
+		);
+
+	public function create(Comment $comment) {
+		$this->validate();
+		$comment->validate();
+
+		if ($this->hasError() || $comment->hasError()) {
+			throw new ValidationException('Invalid thread or comment');
+		}
+
+$db = DB::conn();
```

Comment:
You can also insert the DB::conn() inside the try-catch block :bow: 
This way you can also check if there's a connection error when you tried to connect to the db :dart: